Sources/MapboxCoreNavigation/EventDetails.swift

```diff
@@ -116,6 +116,7 @@ struct NavigationEventDetails: EventDetails {
     var rating: Int?
     var comment: String?
     var userId: String?
+    var appMetadata: [String: String?]? = nil
```

Comment:
`nil` is the default value already, we don't assign it explicitly in other fields, so I guess it would be better to remove it here.